pages/database-management/server-stats.md

```diff
@@ -36,6 +36,7 @@ The result will contain the following fields:
 | vm_max_map_count             | The number of memory-mapped areas that the kernel allows a process to have. If it is unknown, returns -1. </br> For more info, check out [virtual memory section of the docs](/fundamentals/storage-memory-usage#virtual-memory).                                    |
 | memory_res                   | The non-swapped physical RAM memory a task has used, reported by the OS (in B, KiB, MiB, GiB or TiB).                                                                                                                       |
 | peak_memory_res              | Peak RAM memory usage in the system during the whole run.                                        |
+| unreleased_delta_objects     | The current number of objects non-deallocated objects with the information about the changes that write transactions have made, called Delta objects. Delta objects will eventually be deallocated from the system.                        |
```

Comment:
Awkward repetition: "objects non-deallocated objects"

Avoid double negatives such as "non" followed by "de". Perhaps something like "current number of still allocated objects".

Remove "called delta objects".

Please add an answer to the question: "when will they be deallocated?". Saying "eventually" isn't sufficient.

Reply:
corrected and added a link where Delta allocation is explained in detail.